web/xfa_layer_builder.js

```diff
@@ -54,8 +64,8 @@ class XfaLayerBuilder {
       const parameters = {
         viewport: viewport.clone({ dontFlip: true }),
         div: this.div,
-        xfa: this.xfaHtml,
-        page: null,
```

Comment:
Same sort of comment here: is this actually used, and why does the `page` rename only happen here? I would also have expected a similar change in the `XfaLayer` class then, unless I'm missing something here.

Reply:
As above, I think this is completely unused and will fix/remove this.